mysql/resultset_helper.go

```diff
@@ -135,7 +135,12 @@ func BuildSimpleTextResultset(names []string, values [][]interface{}) (*Resultse
 				return nil, errors.Trace(err)
 			}
 
-			row = append(row, PutLengthEncodedString(b)...)
+			if b == nil {
+				// NULL value is encoded as 0xfb here (without additional info about length)
+				row = append(row, 0xfb)
```

Comment:
em, I can't see any difference compared with the origin implementation. seem `row = append(row, 0xfb)` and `row = append(row, []{0xbf}...)` are same.

Reply:
Because func PutLengthEncodedString used yet in https://github.com/siddontang/go-mysql/blob/master/mysql/field.go#L120